cpgserver/src/main/scala/io/shiftleft/cpgserver/route/CpgRoute.scala

```diff
@@ -86,6 +87,7 @@ final class CpgRoute[T: Encoder](cpgProvider: CpgProvider, cpgQueryExecutor: Cpg
         .as[CreateCpgRequest]
         .flatMap(createCpg)
 
+    // TODO discuss with jacob: according to scalac this is unreachable... commenting for now since it probably never worked anyway
```

Comment:
is that the scalac bug mentioned in build.sbt?

Reply:
if you mean this quote from cpgserver/build.sbt: yes
```
// Re-enable fatal warnings once this Scala 2.13 issue is fixed: https://github.com/scala/bug/issues/11457
scalacOptions -= "-Xfatal-warnings"
```